docs/content/sre/tenant-operator/usecases/mattermost.md

```diff
@@ -0,0 +1,35 @@
+### Creating Mattermost Teams for your tenant
+
+Bill wants each tenant to also have their own Mattermost Teams. To Make sure this happens correctly, Bill will first add the `stakater.com/tenant` label to the tenant:
```

Comment:
'To make sure' instead of 'To Make sure'

Reply:
Updated